### DISCUSSION TITLE
llama.cpp & whisper.cpp - now in crowd sourced hardware

### DISCUSSION THREAD

**Discussion by ianscrivener** [ggml-org/llama.cpp#3866] [General]
https://www.crowdsupply.com/useful-sensors/ai-in-a-box

**Comment by ggerganov** [comment]
Cool project, but I'm not sure they are actually using `llama.cpp` and `whisper.cpp`. Rather seems like some custom  lib

**Reply by ianscrivener** [reply]
Oops. I stand corrected.
they are in fact using python...
